server/src/main/java/org/elasticsearch/index/get/GetResult.java

```diff
@@ -222,20 +226,34 @@ public String sourceAsString() {
         return sourceAsMap();
     }
 
+
+    public Map<String, DocumentField> getMetadataFields() {
+        return new HashMap<>(metaFields);
```

Comment:
Do we need to create a new hashmap here or can we just return `metaFields`?

Reply:
done